keps/prod-readiness/sig-scheduling/785.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 785
+beta:
+  approver: "@wojtek-t"
```

Comment:
@wojtek-t 

This is an update bringing the KEP template up to date.  The PRR questions were originally answered back in 1.19.

Reply:
Yeah - I remember that one. Thanks.